src/harness/testcases/grant_testcase.py

```diff
@@ -378,6 +378,53 @@ def test_WINNF_FT_S_GRA_17(self):
     self.assertTrue(response['response']['responseCode'] in (103, 300))
 
   @winnforum_testcase
+  def test_WINNF_FT_S_GRA_19(self):
+    """Frequency range requested overlaps with the frequency range of a FSS incumbent and 
+    CBSD is inside the protection zone of FSS.
+
+    Response Code should be 400"""
+
+    # Register the device
+    device_a = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    request = {'registrationRequest': [device_a]}
+    response = self._sas.Registration(request)['registrationResponse'][0]
+    # Check registration response
+    self.assertEqual(response['response']['responseCode'], 0)
+    cbsd_id = response['cbsdId']
+    del request, response
+
+    # Inject Incumbent Activity with Overlapping Frequency of CBSD
+    fss_zone_0 = json.load(
+      open(os.path.join('testcases', 'testdata', 'fss_zone_0.json')))
+    distance = self.CalculateDistance(device_a['installationParam']['latitude'],
+                                      device_a['installationParam']['longitude'],
+                                      fss_zone_0['deploymentParam'][0]['installationParam']['latitude'],
+                                      fss_zone_0['deploymentParam'][0]['installationParam']['longitude'])
+
+    # CBSD is within the 150km protection zone of FSS
+    self.assertLessEqual(distance, 150)
+    self._sas_admin.InjectFss({'record': fss_zone_0})
```

Comment:
It's possible just to have FSS to create interference, but the CBSD has to be very close to FSS or by putting the FSS and CBSD within a very specific topology. But, I think it's probably best to have FSS with WISP to create Interference.